src/plugins/discover/public/application/main/components/layout/discover_layout.tsx

```diff
@@ -121,8 +118,6 @@ export function DiscoverLayout({
     }
   }, [dataState.fetchStatus]);
 
-  useSavedSearchAliasMatchRedirect({ savedSearch, spaces, history });
```

Comment:
refactored this to `discover_main_app.tsx`, because I don't think there should be a redirect inside this layout component